auto-generated/bfloat16/api-testing/vlmul_ext_v.c

```diff
@@ -0,0 +1,62 @@
+#include <riscv_vector.h>
+#include <stdint.h>
+
+vbfloat16mf2_t test_vlmul_ext_v_b16mf4_b16mf2(vbfloat16mf4_t value) {
```

Comment:
In vlmul_ext_v.c and vlmul_trunc_v.c, b16->bf16.
rvv_intrinsic_gen/templates/misc_op_template.py:109 `args["TYPE1"] = args["TYPE"][0]`